package.json

```diff
@@ -105,14 +108,25 @@
     "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^6.8.1",
     "dotenv": "^16.3.1",
-    "eslint": "^7.27.0",
-    "eslint-plugin-react": "^7.24.0",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-universe": "^12.0.0",
+    "eslint-plugin": "^1.0.1",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jest": "^27.9.0",
+    "eslint-plugin-no-only-tests": "^3.1.0",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-promise": "^6.1.1",
```

Comment:
do we need to add this to the plugin array..?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/719/commits/f41bdf37ba018948b4fe335f58d8270451592b42